vault/token_store.go

```diff
@@ -1472,7 +1472,11 @@ func (ts *TokenStore) handleCreateCommon(
 			// this is a role, add default unless explicitly disabled.
 			if len(finalPolicies) == 0 {
 				finalPolicies = policyutil.SanitizePolicies(parent.Policies, localAddDefault)
-			} else {
+			} else if len(role.DisallowedPolicies) == 0 {
+				// We only do this if role.DisallowedPolicies is empty because
```

Comment:
Is it even possible to get into this branch? Above the case is:
```go
case role != nil && (len(role.AllowedPolicies) > 0 || len(role.DisallowedPolicies) > 0):
```
So I don't think both `role.AllowedPolicies` and `role.DisallowedPolicies` can be 0.